scarb/src/ops/metadata.rs

```diff
@@ -386,6 +399,14 @@ fn wrap_source_id(id: SourceId) -> m::SourceId {
     }
 }
 
+fn wrap_compilation_unit_component_id(
```

Comment:
Why not just `id.to_discriminator().into()`?

Reply:
It was just for consistency with how it's done for `PackageId` and `SourceId`. Can change it